docs/spec/lsm/adr-002.md

```diff
@@ -0,0 +1,362 @@
+
+With a validator-bond factor of 250, for every 1 token a validator self-bonds, that validator is eligible to receive up to two-hundred-and-fifty tokens delegated from liquid staking providers. The validator-bond has no impact on anything other than eligibility for delegations from liquid staking providers.
+
+Without self-bonding tokens, a validator can’t receive delegations from liquid staking providers. And if a validator’s maximum amount of delegated tokens from liquid staking providers has been met, it would have to self-bond more tokens to become eligible for additional liquid staking provider delegations.
+
+### Instantly liquid staking tokens that are already staked
+
+Next, let’s discuss how the LSM makes the adoption of liquid staking more efficient, and can help the blockchain that installs it build strong relationships with liquid staking providers. The LSM enables users to instantly liquid stake their staked tokens, without having to wait the twenty-one day unbonding period. This is important, because a very large portion of the token supply on most Cosmos blockchains is currently staked. Liquid staking tokens that are already staked incur a switching cost in the form of three weeks’ forfeited staking rewards. The LSM eliminates this switching cost.
+
+
+A user would be able to visit any liquid staking provider that has integrated with the LSM and click a button to convert his staked tokens to liquid staked tokens. It would be as easy as liquid staking unstaked tokens.
+
+Technically speaking, this is accomplished by using something called an “LSM share.” Using the liquid staking module, a user can tokenize their staked tokens and turn it into LSM shares. LSM shares can be redeemed for underlying staked tokens and are transferable. After staked tokens are tokenized they can be immediately transferred to a liquid staking provider in exchange for liquid staking tokens - without having to wait for the unbonding period.
+
+
+## Economics:
+
+We expect that eventually governance may decide that the principal agent problems between validators and liquid staking are resolved through the existence of mature liquid staking synthetic asset systems and their associate risk framework. Governance can effectively disable the feature by setting the scalar value to -1 and allow unlimited minting and all liquid delegations to be freely undelegated.
+
+During the transitionary period, this creates a market for liquid shares that may serve to help further decentralize the validator set.
+
+It also allows multiple participants in a validator business to hold their personal stakes in segregated accounts but all collectively contribute towards demonstrating alignment with the safety of the protocol.
+
+
+## Technical Spec
+
+### Modification from ADR-001 
```

Comment:
This section should be more detailed as it contains a lot of very important concepts:

- global liquid staking cap
- validator liquid staking cap
- renames exemption factor to validator bond factor
- 32-length addresses

The first two are already described above. The last two need to be motivated and more clearly described.

Reply:
More detailed spec has been added to `adr.md`